### PR TITLE
JRD integration

### PR DESCRIPTION
**Pull request by sankar-padakula**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/AM-1734
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[N ] No
```

### CODE REVIEW

**Comment by sankar-padakula** [comment]
closing pr to reopen

**Comment by sankar-padakula** [comment]
reopening to create immage

File: charts/am-org-role-mapping-service/values.preview.template.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ java:
     IDAM_S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     CASE_WORKER_REF_APP_URL: http://rd-caseworker-ref-api-aat.service.core-compute-aat.internal
-    JUDICIAL_REF_APP_URL: http://rd-judicial-api-aat.service.core-compute-aat.internal
+    JUDICIAL_REF_APP_URL: http://rd-judicial-api-pr-188.service.core-compute-preview.internal
```
Comment (aashishaggarwal123) [inline charts/am-org-role-mapping-service/values.preview.template.yaml:32]: this should point to AAT now as FTA is disabled.

**Comment by sankar-padakula** [comment]
closing to re-open

File: charts/am-org-role-mapping-service/values.preview.template.yaml (hunk 1 above)
Comment (sankar-padakula) [reply charts/am-org-role-mapping-service/values.preview.template.yaml]: roleback to AAT

**Review by aashishaggarwal123 — APPROVED** [review]

**Review by nitinsahu1 — APPROVED** [review]

**Pull request merged**